app/utils/time.py

```diff
@@ -22,3 +22,9 @@ def is_less_than_days_ago(date_from_db, number_of_days):
 
 def parse_naive_dt(dt):
     return parser.parse(dt, ignoretz=True)
+
+
+def hilite(message):
+    ansi_green = "\033[32m"
+    ansi_reset = "\033[0m"
+    return f"{ansi_green}{message}{ansi_reset}"
```

Comment:
Is this method used for anything?  I'm assuming it's meant for formatting console output, but I don't see it being used anywhere.

Reply:
I like this idea and definitely see the utility!  Perhaps there's a slightly better place for it to call home though.

For lack of a better spot right now, would you be amenable to it moving to the `app/utils.py` file?  This way it's at least in a space that's more general purpose and more easily accessed in other parts of the app with a quick `from app.utils import hilite`. 🙁